### DISCUSSION TITLE
Should the music fade out on death be removed?

### DISCUSSION THREAD

**Discussion by Alzter** [SuperTux/supertux#2037] [General]
It seems weird for the music to fade out when Tux dies, considering it just resumes anyway. Should the fade out/in be removed?

**Comment by Alzter** [comment]
Made a PR for this #1472

**Comment by busybody42** [comment]
keep it.

**Comment by Alasdairbugs** [comment]
it should be removed, its clunky, and for levels where you die a lot, it just gets annoying and buggy. it sometimes just cuts out all together.

**Comment by Ordoviz** [comment]
I like the fade-out/fade-in if you wait during the die animation. But I always press <kbd>Esc</kbd> to skip the animation which instantly stops the music. Because of that I support the removal.

**Comment by busybody42** [comment]
I think it should instantly skip the fade out if you skipped the death cutscene...

**Comment by tux314159** [comment]
And maybe also let the music continue after a restart?

**Comment by MrIceblock** [comment]
Keep it, IMO it would be nice if the music fades out upon death and fade back in upon revival.

**Comment by CPa129** [comment]
Don't keep it if the player died 5 times or six times because as the fade out will slow the game, which will be very annoying for the player.
